src/Monitoring/Monitoring.ArcadeServices/dashboard/arcade-services/arcadeAvailability.dashboard.json

```diff
@@ -65,7 +65,7 @@
         "frequency": "1m",
         "handler": 1,
         "message": "Internal Pool Provider Average Response Time is high",
-        "name": "Average Response Time alert",
+        "name": "Internal Pool Provider Average Response Time",
         "noDataState": "keep_state",
```

Comment:
While you're here, want to set `"executionErrorState": "keep_state"` for these alerts?

Reply:
Done. :-)